packages/compass-sidebar/src/components/sidebar/sidebar.jsx

```diff
@@ -232,7 +237,7 @@ const mapStateToProps = (state) => ({
   isDataLake: state.isDataLake,
   isGenuineMongoDB: state.isGenuineMongoDB,
   isGenuineMongoDBVisible: state.isGenuineMongoDBVisible,
-  isCSFLEConnection: state.isCSFLEConnection,
+  csfleMode: state.csfleMode,
```

Comment:
Unused prop? It looks like we pull `csfleMode` from `instance` in `sidebar-instance.jsx`

Reply:
Yup, thanks for catching 👍